Manito/Manito/Network/EndPoint/LetterEndPoint.swift

```diff
@@ -51,12 +52,12 @@ enum LetterEndPoint: EndPointable {
 
     func getURL(baseURL: String) -> String {
         switch self {
-        case .dispatchLetter(let roomId,_,_):
+        case .dispatchLetter(let roomId,_,_,_):
```

Comment:
p4;

```suggestion
        case .dispatchLetter(let roomId, _, _, _):
```
혹시 요렇게 안될까요...??

Reply:
반영해뒀습니다.
[d4db52a](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/396/commits/d4db52acbf114e0eaf8eb6987f8505e2963c3048)